### DISCUSSION TITLE
WalletConnect redirects to app store when opened in brave browser incognito mode

### DISCUSSION THREAD

**Discussion by way2rach** [WalletConnect/walletconnect-monorepo#3394] [Developer Support]
Routing to app store when opened in incognito mode in brave browser

**Devices tested:**
iPhone (brave - incognito mode)

**App to try:**
https://app.uniswap.org/#/swap

**Steps to repro:**
Install BitPay app or MetaMask app and then go to uniswap in incognito mode, try to connect using wallet link, it will redirect to app store

**Comment by brennanfife** [comment]
Adding to this, this issue seems to only be with the Brave mobile browser. Some wallets (i.e. imToken) work perfectly fine, but others (i.e. Rainbow Wallet) always direct to the app store.

**Comment by finessevanes** [comment]
@way2rach @brennanfife is this still an issue?

**Comment by way2rach** [comment]
yes

**Comment by glitch-txs** [comment]
Have you tried this on v2 migration?
